definitions/npm/tape_v4.5.x/flow_>=v0.25.x/tape_v4.5.x.js

```diff
@@ -11,7 +11,7 @@ declare type tape$TestOpts = {
 };
 
 
-declare type tape$TestCb = (t: tape$Context) => void;
+declare type tape$TestCb = (t: tape$Context) => void | Promise<mixed>;
```

Comment:
It'd be better just to write `mixed` instead and just accept anything since it doesn't make a difference to tape (AFAIK)